src/rgw/rgw_rest.cc

```diff
@@ -644,6 +644,7 @@ void dump_trans_id(req_state *s)
 {
   if (s->prot_flags & RGW_REST_SWIFT) {
     dump_header(s, "X-Trans-Id", s->trans_id);
+    dump_header(s, "X-Openstack-Request-Id", s->trans_id);
```

Comment:
Great! Swift does that in exactly the same way:
* https://github.com/openstack/swift/blob/master/swift/common/middleware/catch_errors.py#L51-L55,
* https://github.com/openstack/swift/blob/master/swift/common/middleware/catch_errors.py#L47-L48,
* https://github.com/openstack/swift/blob/master/swift/proxy/controllers/base.py#L1773-L1774.

Commit bringing support for `X-Openstack-Request-Id" in Swift is https://github.com/openstack/swift/commit/d0fe922904c8016267e89c97bef16052a2c3bb7e.